examples/universal-cnf/vppagent/pkg/config/config_endpoints.go

```diff
@@ -58,7 +58,7 @@ func NewProcessEndpoints(backend UniversalCNFBackend, endpoints []*nseconfig.End
 			OutgoingNscName:    nsconfig.OutgoingNscName,
 			AdvertiseNseLabels: labelStringFromMap(e.Labels),
 			MechanismType:      memif.MECHANISM,
-			IPAddress:          e.VL3.IPAM.PrefixPool,
+			IPAddress:          e.VL3.IPAM.DefaultPrefixPool,
```

Comment:
You shouldn't need to change this line.  You should only have to have changed how the IPAM endpoint param gets set in the nse.go: 
```
ipamEp := endpoint.NewIpamEndpoint(&common.NSConfiguration{
		IPAddress: prefixPool,
	})
```

Reply:
This change is necessary due ti changes made in k8s-vnet repo. It is just a name refacoring and it wont compile otherwiseotherwise